client/stylesheets/_responsive.scss

```diff
@@ -373,10 +461,15 @@
 }
 
 
-@media (max-width: 357px) { 
-    // .footerLeft ul:first-child li:nth-child(6) {
-    //     margin-top: 20px !important;
-    // }
+@media (max-width: 342px) { 
+    .homeCardLi {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `homeCardLi` should be written in lowercase with hyphens